### DISCUSSION TITLE
Youtube controls, Feedback button are disabled

### DISCUSSION THREAD

**Discussion by cescapa** [OpenMined/courses#102] [Feedback] [answered]
I am having a similar issue to https://github.com/OpenMined/courses/discussions/97 ; since that discussion is locked I am unable to update it. 

The embedded __Youtube screen does not show the timeline or the mouse controls__, even after hovering and pausing/restarting the video. I am running Chrome Version 87.0.4280.88 (Official Build) (arm64). 

The consequence is that __the student does not know how long each lecture runs for__; also she will not be able to change the screen resolution or select the subtitle language. 

The __[Youtube keyboard shortcuts](https://support.google.com/youtube/answer/7631406?hl=en)__ work normally, so it is possible to turn on default subtitles by clicking "c", and change the speed by clicking "<" or ">". There does not appear to be a shortcut to change the resolution. 

The __feedback__ button does not do anything. 

I have Ad Block but it is disabled.

**Comment by mcleonard** [comment]
If you're on Desktop you can scroll down a bit show the controls.

<img width="1087" alt="Screen Shot 2021-01-03 at 10 32 27 AM" src="https://user-images.githubusercontent.com/1200974/103485954-0ac17c00-4daf-11eb-9070-b49dbe3bf7ab.png">

And yeah, looks like the feedback button isn't working for me right now

**Reply by cescapa** [reply]
Thanks Mat, the controls don't appear even after scrolling down. However, in an Intel machine, Chrome is behaving normally. I suspect that the Youtube missing controls issue is unrelated to your courseware and could be related to the ARM64 version of Chrome.  

Since there is already another issue for the feedback button I suggest that you go ahead and close this one.

**Comment by mcleonard** [comment, marked as answer]
There's an issue for the feedback button here: https://github.com/OpenMined/openmined/issues/76
